app/org/sagebionetworks/bridge/services/email/WithdrawConsentEmailProvider.java

```diff
@@ -45,9 +45,11 @@ public MimeTypeEmail getMimeTypeEmail() throws MessagingException {
 
         final String sendFromEmail = getFormattedSenderEmail();
         builder.withSender(sendFromEmail);
-
-        Set<String> emailAddresses = BridgeUtils.commaListToOrderedSet(getStudy().getConsentNotificationEmail());
-        builder.withRecipients(emailAddresses);
+        
+        if (getStudy().getConsentNotificationEmail() != null) {
+            Set<String> emailAddresses = BridgeUtils.commaListToOrderedSet(getStudy().getConsentNotificationEmail());
+            builder.withRecipients(emailAddresses);
+        }
```

Comment:
If there is no consent notification email, then there are no recipients for this email. What happens in this case?

There's also no corresponding unit test for this change.